yjit/src/stats.rs

```diff
@@ -141,6 +141,19 @@ macro_rules! make_counters {
     }
 }
 
+/// Macro to increase a counter by name and count
+macro_rules! add_counter {
```

Comment:
```suggestion
macro_rules! counter_incr_by {
```

Or, `counter_add_to`

`add_counter` sounds like you're adding a new counter.

Reply:
Agreed, and your version sounds better. Fixed in https://github.com/ruby/ruby/pull/7397/commits/6e9e9cf8c27f81d0647942f4d011b9deff51434a.